### PR TITLE
Ike is a crybaby - raises plasmaman speed to comp for the downgrade of regular speed

### PR DESCRIPTION
**Pull request by Qustinnus**
:cl: Qustinnus
tweak: tweaks plasmaman speed
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -147,7 +147,7 @@
 		if(/obj/item/projectile/energy/florayield)
 			H.nutrition = min(H.nutrition+30, NUTRITION_LEVEL_FULL)
 	return
-	
+
```
Comment (Vivalas) [inline code/modules/mob/living/carbon/human/species_types.dm]: Why is there a random linebreak here?

Comment (ike709) [inline code/modules/mob/living/carbon/human/species_types.dm]: Blame the oldcoders.

**Comment by Vivalas** [comment]
@Qustinnus Remove that line break please.

**Comment by Qustinnus** [comment]
we reverted speed so fuck this pr :^) bye @ike709

**Pull request closed without merging**